chain/network/src/network_protocol/mod.rs

```diff
@@ -325,6 +324,11 @@ pub struct SyncAccountsData {
     pub incremental: bool,
 }
 
+#[derive(PartialEq, Eq, Clone, Debug)]
+pub struct Disconnect {
+    pub allow_reconnect: bool,
```

Comment:
Please document how allow_reconnect should be interpreted by the node.
AFAIR you were planning to put an enum with reasons here instead. Has something changed?

Reply:
Originally I was sharing the exact `PeerActor::ClosingReason` which is unnecessary (it seems silly for example to share the ban reason with a bad actor). Is there any information you think makes sense to include here in addition to `allow_reconnect`?